### PR TITLE
Not cleaning before deploy

### PR DESCRIPTION
**Pull request by KevinJCross**
### Change description ###

Still not deploying source
<br>**Does this PR introduce a breaking change?** 
```
[ ] Yes
[x ] No
```

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,11 +13,14 @@
     "clean": "rimraf lib && rimraf coverage",
     "lint": "tsc --noEmit && tslint --project tsconfig.json",
     "prebuildOnly": "yarn clean && yarn format && yarn lint && echo Using TypeScript && tsc --version",
-    "build": "tsc --pretty",
+    "build": "rimraf lib && tsc --pretty",
     "test": "jest",
     "coverage": "jest --coverage",
     "watch": "yarn build --watch",
-    "watch:test": "jest --watch"
+    "watch:test": "jest --watch",
```
Comment (KevinJCross) [inline package.json]: not sure why these changes are changes ... they are already there ?

Comment (timja) [reply package.json]: comma change

Comment (KevinJCross) [reply package.json]: also a local unpushed merge ... sigh.
Not my day for Node.
